openapi/platform/platform.yaml

```diff
@@ -2062,13 +2056,10 @@ paths:
           name: 'filter[trigger]'
           description: Returns automations which have a trigger matching the provided value.
         - schema:
-            type: array
-            items:
-              type: string
-            example: default
+            type: string
           in: query
-          name: 'filter[marketIds]'
-          description: Returns automations matching the provided market IDs.
+          name: 'filter[namespace]'
+          description: Returns automations for the provided namespace
```

Comment:
Partners do not know what a namespace is. We should clarify that it is an account id or partner id.